### PR TITLE
Sep 007

### PR DESCRIPTION
**Pull request by cjmyers**
Completed combinatorial derivations extension.

### CODE REVIEW

**Comment by cjmyers** [comment]
This pull request fixes #25 and #18.

File: lib/SBOLDocument.js [hunk 1]
```diff
@@ -52,11 +52,15 @@ var ModuleDefinition = require('./ModuleDefinition'),
...
     GenericLocation = require('./GenericLocation'),
     GenericTopLevel = require('./GenericTopLevel'),
+    CombinatorialDerivation = require('./CombinatorialDerivation'),
+    VariableComponent = require('./VariableComponent'),
     ProvActivity = require('./ProvActivity'),
     ProvAssociation = require('./ProvAssociation'),
     ProvAgent = require('./ProvAgent'),
     ProvPlan = require('./ProvPlan'),
     ProvUsage = require('./ProvUsage')
+    CombinatorialDerivation = require('./CombinatorialDerivation'),
```
Comment (nathanwilk7) [inline lib/SBOLDocument.js:62]: no big deal, but we should remove the extra imports of Combinatorial and Variable

File: lib/load/rdf/loadRDF.js [hunk 2]
```diff
@@ -50,12 +50,16 @@ var SBOLDocument = require('../../SBOLDocument'),
...
     loadSequenceConstraint = require('./loadSequenceConstraint'),
+    loadCombinatorialDerivation = require('./loadCombinatorialDerivation'),
+    loadVariableComponent = require('./loadVariableComponent'),
     loadProvActivity = require('./loadProvActivity'),
     loadProvAssociation = require('./loadProvAssociation'),
     loadProvUsage = require('./loadProvUsage'),
     loadProvPlan = require('./loadProvPlan'),
     loadProvAgent = require('./loadProvAgent'),
     loadGenericTopLevel = require('./loadGenericTopLevel');
+    loadCombinatorialDerivation = require('./loadCombinatorialDerivation'),
```
Comment (nathanwilk7) [inline lib/load/rdf/loadRDF.js:61]: Same double import

**Comment by cjmyers** [comment]
Likely introduced due to the merge conflicts when I rebased with master.  Please do feel free to fix this.

**Pull request merged**